### PR TITLE
Added snips transposing a 2D Array

### PR DESCRIPTION
**Pull request by coderSuren**
Completes #26 
Added code snips for transposing a 2D array in 5 languages (C++, python, java, javascript, bash)

### CODE REVIEW

File: src/Transpose_array.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    return B;
+}
+
+```
+
+
+### JavaScript
+```js
+function transposeArray(A) {
+    var B = Array(N);
```
Comment (DevTMK) [inline src/Transpose_array.md:65]: Check your code once in js, cpp, bash mate

**Review by DevTMK — CHANGES_REQUESTED** [review]

File: src/Transpose_array.md (hunk 1 above)
Comment (coderSuren) [reply src/Transpose_array.md]: Sorry, I forget to change the value of 'N' in javascript, changed it now. I also checked code in C++ and bash.

File: src/Transpose_array.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+}
+
+```
+
+
+### JavaScript
+```js
+function transposeArray(A) {
+    var B = Array(A.length);
+    for(i=0; i<N; i++){
```
Comment (DevTMK) [inline src/Transpose_array.md:66]: still issues, check and run all snippets once please

**Review by DevTMK — CHANGES_REQUESTED** [review]

**Review by DevTMK — APPROVED** [review]
Thanks @coderSuren, Merging

**Pull request merged**